Evenly spread queriers across available nodes (#6415)
grafana/loki: docs/sources/upgrading/_index.md

#### Evenly spread queriers across kubernetes nodes

We now evenly spread queriers across the available kubernetes nodes, but allowing more than one querier to be scheduled into the same node.

> If you want to keep running up to one querier per node, set `$._config.querier.use_topology_spread` to false.

Reviewer: ```suggestion
If you want to run at most a single querier per node, set `$._config.querier.use_topology_spread` to false.
```

Author: Done
